packages/wallets/package.json

```diff
@@ -23,6 +23,7 @@
         "generate:docs": "typedoc"
     },
     "dependencies": {
+        "@crossmint/client-sdk-base": "workspace:*",
```

Comment:
Needed for wallet error codes

Reply:
hmm that would require moving the `CrossmintSDKError` from `client-base` to `common-base` as well

Reply:
looks good https://github.com/Crossmint/crossmint-sdk/pull/1009/commits/b2cf7ea221287e05e7914ed05270ab8ca185d990